### PR TITLE
Indirect Update Documentation

### LINKED ISSUE #24666: Indirect Documentation needs updating
### Description
Much of the Indirect documentation has become outdated. This includes the documentation for Indirect Data Analysis, Data Reduction, Corrections, Bayes, Simulation, Diffraction and Tools. 

The formula for MSDGauss also needs updating.

### PR DESCRIPTION
**Pull request by robertapplin**
**Description of work.**
This PR tries to update nearly ALL of the Indirect documentation. (Not the documentation in the old wiki which is going to be moved over at some point)

**To test:**
Read through the documentation. Check for typos.

The QENSFitSequential diagram should now have OutputParameterWorkspace and OutputWorkspaceGroup connected within the diagram.

Fixes #24666 #24541 

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by TTitcombe — CHANGES_REQUESTED** [review]
Documentation seems a lot more thorough and clear than previously. However I've been really nitpicky and highlighted a few issues.

**Review by TTitcombe — APPROVED** [review]
This PR is largely changes to documentation. The documentation is clear and is a big improvement

File: Framework/PythonInterface/plugins/functions/MsdGauss.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def function1D(self, xvals):
         msd = self.getParameterValue("Msd")
 
         xvals = np.array(xvals)
-        intensity = height * np.exp(-msd * xvals**2)
+        intensity = height * np.exp((-msd * xvals**2)/6)
```
Comment (martyngigg) [inline Framework/PythonInterface/plugins/functions/MsdGauss.py]: This looks like more than a documentation update? Was the formula here incorrect before?

Comment (robertapplin) [reply Framework/PythonInterface/plugins/functions/MsdGauss.py]: Yes the formula was incorrect - I noticed this when updating the MsdGauss documentation. Sanghamitra has informed me that the MSDPeters and MSDYi functions should reduce to the MSDGauss formula given certain conditions, and so this means a 1/6 factor needs to be added to the exponent.

Comment (martyngigg) [reply Framework/PythonInterface/plugins/functions/MsdGauss.py]: :+1:

**Pull request merged**